controlplane/migrations/0087_deep_lake.sql

```diff
@@ -0,0 +1,7 @@
+DO $$ BEGIN
+ CREATE TYPE "websocket_subprotocol" AS ENUM('auto', 'graphql-ws', 'graphql-transport-ws');
+EXCEPTION
+ WHEN duplicate_object THEN null;
+END $$;
+--> statement-breakpoint
+ALTER TABLE "subgraphs" ADD COLUMN "websocket_subprotocol" "websocket_subprotocol";
```

Comment:
Should we set this to NOT NULL default "auto"?

Reply:
we could, the reason i didnt is because there are a few entries on the db which have the subscription protocol as sse